addons/l10n_in_ewaybill/models/ir_attachment.py

```diff
@@ -17,4 +17,3 @@ def _unlink_except_government_document(self):
             for attachment in self
         ):
             raise UserError(_("You can't unlink an attachment that you received from the government"))
-        return super()._unlink_except_government_document()
```

Comment:
Then we would miss the `super` call when the EDI module is installed
As a patch, you could just rename the method here
So it will not be considered as an override